### DISCUSSION TITLE
Compatability with Databricks runtime 13.3 ML

### DISCUSSION THREAD

**Discussion by carlosram17** [NVIDIA/spark-rapids#10091] [Q&A] [answered]
Hello,

Do you know when spark-rapids will be available on Databricks runtime 13.3 ML ?

**Comment by gerashegalov** [comment, marked as answer]
Hi @carlosram17, support for DBR 13.3 did not make the 23.12 release #9890 but should go into the 24.02 release in February

**Comment by carlosram17** [comment]
Thanks for the info :)
